README.md

```diff
@@ -110,8 +110,14 @@ so it runs anywhere Git and Bash runs.
 * __Autocompletion for [Fish shell](http://fishshell.com)__
 
     ```
-    mkdir -p ~/.config/fish/completions/
-    curl -o ~/.config/fish/completions/git.fish http://raw.githubusercontent.com/Originate/git-town/master/autocomplete/git.fish
+    $ git town install-fish-autocompletion
+    ```
+
+    or manually
+
+    ```
+    $ mkdir -p ~/.config/fish/completions/
+    $ curl -o ~/.config/fish/completions/git.fish http://raw.githubusercontent.com/Originate/git-town/master/autocomplete/git.fish
```

Comment:
Okay, removed the manual steps. I still want to at least mention how to install the autocompletions. If I would remove this here, where would this live then in your opinion?